packages/stores/src/holochain.ts

```diff
@@ -453,7 +453,8 @@ export function liveLinksStore<
   baseAddress: BASE,
   fetchLinks: () => Promise<Array<Link>>,
   linkType: LinkTypeForSignal<S>,
-  pollIntervalMs: number = DEFAULT_POLL_INTERVAL_MS
+  firstFetchLinks?: () => Promise<Array<Link>>,
```

Comment:
Oh I just realized, to make this a non-breaking change it would need to be added as the last parameter, i.e. after `pollIntervalMs`. Not a big deal either way but maybe a nice to have...

Reply:
Ok, I put it there because you can't have a ? item after an initialized item, but I converted that to just having a default of `undefined` and it worked.